samples/client/petstore/scala-sttp/README.md

```diff
@@ -0,0 +1,121 @@
+# scala-akka-petstore-client
```

Comment:
here and below, names should be changed accordingly to `sttp`

Reply:
fixed, thanks